src/pages/SearchPage.js

```diff
@@ -2,26 +2,35 @@ import styled from "styled-components/macro";
 import PropTypes from "prop-types";
 import ConcertCard from "../components/ConcertCard";
 import Heading from "../components/Header";
+import React from "react";
+import SearchInput from "../components/SearchInput";
 
 SearchPage.propTypes = {
   pageName: PropTypes.string,
   concerts: PropTypes.array,
   onClick: PropTypes.func,
   onNavigate: PropTypes.func,
+  setCity: PropTypes.func,
 };
 
-export default function SearchPage({ pageName, onNavigate, concerts }) {
+export default function SearchPage({
+  pageName,
+  onNavigate,
+  concerts,
+  setCity,
+}) {
   return (
     <>
       <Heading pageName={pageName} />
```

Comment:
`Heading` should not "know" anything about pages.
```suggestion
      <Heading text={pageName} />
```